GiTracker/Controls/UserControl.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace GiTracker.Controls
+{
+    public class UserControl : Grid
```

Comment:
UserControl is a common name, it is present in many frameworks, so please rename.